docs/html/_sources/usage.rst.txt

```diff
@@ -348,6 +348,12 @@ where
     to avoid shell substitution.
   * Names of specific tar archives to list all files within these tar archives.
 
+.. warning::
+    Running ``zstash ls`` creates a cahce (by default named ``zstash``).
```

Comment:
"cahce" is mispelled here, but spelled correctly in the rst file below. You probably need to rerun 'make html'